src/runtime/transformers/path-meta.ts

```diff
@@ -67,7 +67,8 @@ const isPartial = (path: string): boolean => path.split(/[:/]/).some(part => par
  * @returns generated slug
  */
 export const generatePath = (path: string, { forceLeadingSlash = true } = {}): string => {
-  path = path.split('/').map(part => slugify(refineUrlPart(part), { lower: true })).join('/')
+  const { content } = useRuntimeConfig().public
```

Comment:
We should avoid using Nuxt generated helpers (like `useRuntimeConfig`) in transformers. Transformers and in general the content parser could be able to work outside of Nuxt Context.

There is already several usecases of parser outside of Nuxt.

`respectPathCase`  could be passed in second argumant of this function and transformer itseld